### PR TITLE
Add comment icon and "You commented" translations to activities

### LINKED ISSUE #475: Remove left margin from files tabview
Left margin causes inconsistency of different tabs in the files sidebar:

The activity tab _with_ margin:
bildschirmfoto von 2016-03-10 08-58-53
Other tabs don't have it:
bildschirmfoto von 2016-03-10 08-59-04

@nickvergessen @jancborchardt is that intended?

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix https://github.com/owncloud/activity/issues/475

@ChristophWurst @blizzz 

@karlitschek just some cosmetics, but I'd suggest a backport to 9.0

### CODE REVIEW

**Comment by karlitschek** [comment]
backport makes sense :+1:

**Comment by rullzer** [comment]
:+1:

File: apps/comments/activity/extension.php [hunk 1]
```diff
@@ -177,6 +183,21 @@ protected function translateLong($text, IL10N $l, array $params) {
...
+	 * Check if the author is the current user
+	 *
+	 * @param string $user Parameter e.g. `<user display-name="admin">admin</user>`
+	 * @return bool
+	 */
+	protected function authorIsCurrentUser($user) {
+		try {
+			return strip_tags($user) === $this->activityManager->getCurrentUserId();
+		} catch (\UnexpectedValueException $e) {
+			// FIXME this is awkward, but we have no access to the current user in emails
```
Comment (rullzer) [inline apps/comments/activity/extension.php]: hehe :)

**Comment by jancborchardt** [comment]
Great stuff, thank you @nickvergessen!

**Pull request merged**